src/main/java/com/github/argon/sos/moreoptions/ui/panel/BoostersPanel.java

```diff
@@ -85,67 +76,19 @@ public BoostersPanel(List<Entry> boosterEntries, String configFilePath) {
 
         }
 
-        int width = 0;
-
-        Map<String, Slider> elements = new HashMap<>();
-        Map<String, List<List<? extends GuiSection>>> mapRows = new HashMap<>();
-        List<List<? extends GuiSection>> rows = new ArrayList<>();
-
-        for(String keyDef: boosterDefinitions.keySet()) { //it's stupid, but it works
-            List<List<? extends GuiSection>> innerList = new ArrayList<>();
-            boosterDefinitions.get(keyDef).entrySet()
-                    .stream().sorted(Comparator.comparing(entry -> entry.getValue().getLabelDefinition().getTitle()))
-                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue,
-                            (oldValue, newValue) -> oldValue, LinkedHashMap::new)).forEach((key, definition) -> {
-                BuildResult<List<GuiSection>, Slider> buildResult = BoosterSliderBuilder.builder()
-                        .definition(definition)
-                        .build();
-                rows.add(buildResult.getResult());
-                elements.put(key, buildResult.getInteractable());
-                innerList.add(buildResult.getResult());
-
-            });
-            mapRows.put(keyDef, innerList);
-        }
-
-        width = UiUtil.getMaxCombinedColumnWidth(rows);
-
-        for(String key: boosterDefinitions.keySet()) {
-            GHeader header = new GHeader(key, UI.FONT().H2);
-            header.hoverInfoSet(key);
+        BuildResult<GScrollRows, Map<String, Slider>> buildResult = BoosterSlidersBuilder.builder()
+                .displayHeight(500)
+                .translate(boosterDefinitions)
+                .build();
 
-            renderobjs.add(header);
+        GScrollRows gScrollRows = buildResult.getResult();
+        sliders = buildResult.getInteractable();
 
-            List<List<? extends GuiSection>> innerRows = mapRows.get(key);
-            List<Integer> maxWidths = UiUtil.getMaxColumnWidths(innerRows);
-
-
-            int finalWidth = width;
-            List<GridRow> gridRows = innerRows.stream()
-                    .map(columns -> {
-                        GridRow gridRow = new GridRow(columns);
-                        gridRow.initGrid(maxWidths, finalWidth);
-
-                        if (innerRows.indexOf(columns) % 2 == 0) {
-                            gridRow.background(COLOR.WHITE15);
-                        }
-
-                        return gridRow;
-                    })
-                    .collect(Collectors.toList());
-
-            renderobjs.addAll(gridRows);
+        GHeader disclaimer = new GHeader("High values can slow or even crash your game");
+        disclaimer.hoverInfoSet(new INFO("In case of a crash", "Delete configuration file in:" + configFilePath));
```

Comment:
I guess this hover tooltip is obsolete now; this is from before I added a crash handler ^^